posts/2021-08-18_dingwall_to_cromarty/content.html

```diff
@@ -0,0 +1,50 @@
+<h3><strong>T</strong>here are many ways to go from Dingwall to Cromarty. Except for walking, we tried most of them this summer.</h3>
+
+<h3><strong>B</strong>y kayak (6 hours)</h3>
+
+<p>This paddle in the Cromarty firth is about 30 km, which is far, but we figured with an outgoing tide it should be doable. 
+We started quite early with a mocha (Leigh) and chai latte (Annette) from a super cute coffee place our landlords had attended us of. The owner is Canadian, so when we went scouting the place out a few days earlier we had chatted a bit about living in Scotland. Dingwall (which name dates back from the Viking days) has a canal that connects the town to the firth. It was built in 1819 but not really a commercial success, so abandoned in the 1880s when a railway provided a better connection to the town. The canal is still there, but even by kayak it can be only paddled at higher tides, which was not a problem for us because we were leaving at high tide anyway to not have to paddle against an upcoming tide. Paddling out of the canal was fun, would the wooden structures that we saw at the end of the canal really be 200 years old? </p>
```

Comment:
```suggestion
We started quite early with a mocha (Leigh) and chai latte (Annette) from a super cute coffee place our neighbours had alerted us of. The owner is Canadian, so when we went scouting the place out a few days earlier we had chatted a bit about living in Scotland. Dingwall (which name dates back from the Viking days) has a canal that connects the town to the firth. It was built in 1819 but not really a commercial success, so abandoned in the 1880s when a railway provided a better connection to the town. The canal is still there, but even by kayak it can be only paddled at higher tides, which was not a problem for us because we were leaving at high tide anyway to not have to paddle against an upcoming tide in the firth. Paddling out of the canal was fun, would the wooden structures that we saw at the end of the canal really be 200 years old? </p>
```